generator/README.md

```diff
@@ -24,8 +26,8 @@ Once [Java v1.8](https://www.java.com/en/download/manual.jsp) is installed you c
 **Note:** Do not include a trailing \ in directory paths
 
 ## Examples
-* `java -jar generator.jar generate profile.json profile.csv`
```

Comment:
Would it be better to keep this as a concise example, but maybe mention that there are some example profiles that can be used?